### ISSUE TITLE
2.7 Bug collection thread

### ISSUE THREAD

**Issue by G1DE30N** [Akebi-Group/Akebi-GC#46]
**Bug Table**

Player:
- ~~Godmode - Works for block Fall Damage but doesn't work for all other sources of damage (mob attacks, damage traps, effects of elements etc.~~
- ~~Noclip - Most of all works fine but NoAnimation Func doesn't do anything at all.~~

World:
- ~~Auto Treasure - Wont open chests.~~
- ~~Auto Talk - Auto Select dialog doesn't work.~~
- ~~Auto Tree Farm - Doesn't Work Entirely.~~
- ~~Auto Cook  - Doesn't Work Entirely.~~
- ~~Kill Aura - Doesn't Work Entirely.~~

Teleport:
- ~~Any type of Teleporting Crashes the game. (Or for someone simple won't work at all)~~

Visuals:
~~FPS Unlock - Crashes the game.~~

Interactive Map:
~~- Doesn't work entirely except radar, alll icons messed up and move on screen while you scroll map with cursor. Sometimes icons doesn't shows at all.~~

==================================================

MAYBE ONLY FOR ME: (@CallowBlack, Just a note for info, Don't focus your attention too much on this, because so far there is no confirmation from other players here. )
 - UI messed up entirely immediately after starting the game with no matter what cheat funcs active or not. The game stupidly starts immediately with a broken interface. I thinks devs f**cked up render engine in new update even stronger than in the previous version. Perhaps that's why the interactive map broke down. But again, these are all my theories, in general I'm thinking that most likely the gui library itself that you use is the root of this evil. So many mysteries.

==================================================

**Here are not tested for functionality features:**

Player:
- ~~Attack Effects (_Multi-Target_, _Radius_)~~
- ~~Cooldown Effects (_No Sprint Cooldown_, _Instant Bow Charge_)~~

World:
- ~~Auto Destroy Objects (_Shields_, _Doodads_)~~
- ~~Auto Treasure (_Leyline_, _Search Points_, _Quest Interacts_, _Others_)~~
- Fishing (_Delay_, _Recast Rod_, _Recast Delay_)
- ~~Mob Vacuum (_Animals_)~~

**Potentialy harmless (as Taiga said):**
A bunch of error codes in startup and reminder about chinesse client.

**_If you find anything except this list, please, provide it below and i add that info there. 
Thats helps to Callow, Taiga and other contributors fix this ASAP._**

**Comment by monado1110** [comment]
what features are working on 2.7 ?

**Comment by G1DE30N** [comment]
As i see Miltihit Works, another not tested yet.

**Comment by Taiga74164** [comment]
is it only when enemies hit you? or it doesnt work when you jump from high places aswell?

**Comment by LokeYourLord** [comment]
Oh so if I build Akebi now it already works for 2.7?

**Comment by Taiga74164** [comment]
> Oh so if I build Akebi now it already works for 2.7?
> 
> Edit: Off-topic I know but I kinda wanna know.

yes. please report any issues you find

**Comment by LokeYourLord** [comment]
> > Oh so if I build Akebi now it already works for 2.7?
> > Edit: Off-topic I know but I kinda wanna know.
> 
> yes. please report any issues you find

Alright certainly will do, but how "stable" or "safe" is it compared to 2.6?

**Comment by G1DE30N** [comment]
> is it only when enemies hit you? or it doesnt work when you jump from high places as well? (can't test myself at the moment, sorry)

Only with MOBS. Falling fine.

**Comment by G1DE30N** [comment]
No animation in noclip don't work.

**Comment by Taiga74164** [comment]
> > > Oh so if I build Akebi now it already works for 2.7?
> > > Edit: Off-topic I know but I kinda wanna know.
> > 
> > 
> > yes. please report any issues you find
> 
> Alright certainly will do, but how "stable" or "safe" is it compared to 2.6?

safety - same as before
stable - not sure since we tried our best to update as fast as possible. expect a stable build after all issue has been addressed.

**Comment by G1DE30N** [comment]
As i mentioned in another issue If you try use Teleport on map (T key) game instantly crashes. Don't use guys.

**Comment by LokeYourLord** [comment]
> > > > Oh so if I build Akebi now it already works for 2.7?
> > > > Edit: Off-topic I know but I kinda wanna know.
> > > 
> > > 
> > > yes. please report any issues you find
> > 
> > 
> > Alright certainly will do, but how "stable" or "safe" is it compared to 2.6?
> 
> safety - same as before stable - not sure since we tried our best to update as fast as possible. expect a stable build after all issue has been addressed.

Alright cool, just wanted a rough estimate which you gave me now. I'll report any issues I'll find as detailed as I can. If I can fix some of them with a Pull Request I will, although I certainly am not as experienced as you guys.

**Comment by G1DE30N** [comment]
Kill Aura won't works in any modes.

**Comment by LokeYourLord** [comment]
> -snip-

@G1DE30N You might wanna change the title thread to "2.7 Bug collection thread" or something like that.

**Comment by G1DE30N** [comment]
> > -snip-
> 
> @G1DE30N You might wanna change the title thread to "2.7 Bug collection thread" or something like that.

Yep i do

**Comment by LokeYourLord** [comment]
"Godmode bug and another not worked funcs topic :D"
Grammatically correct: Godmode bug and other non-functioning features :D

**Comment by G1DE30N** [comment]
> "Godmode bug and another not worked funcs topic :D" Grammatically correct: Godmode bug and other non-functioning features :D

Just jerking around. I Change name and first post which i updates after another test and discoveries)))0

**Comment by LokeYourLord** [comment]
> > "Godmode bug and another not worked funcs topic :D" Grammatically correct: Godmode bug and other non-functioning features :D
> 
> Just jerking around. I Change name and first post which i updates after another tests and discoveries)))0

Lol, gotcha. Anyways yeah let's post all bugs we find here in a collective list.

**Comment by LokeYourLord** [comment]
I got one @Taiga74164 :

![image](https://user-images.githubusercontent.com/5807595/171086495-2b92b09d-d504-494c-bbcf-707f98b1803c.png)

**Comment by G1DE30N** [comment]
> I got one @Taiga74164 :
> 
> ![image](https://user-images.githubusercontent.com/5807595/171086495-2b92b09d-d504-494c-bbcf-707f98b1803c.png)
> 
> I am not using the Chinese version lul.

I posted that just ignore.

**Comment by LokeYourLord** [comment]
> > I got one @Taiga74164 :
> > ![image](https://user-images.githubusercontent.com/5807595/171086495-2b92b09d-d504-494c-bbcf-707f98b1803c.png)
> > I am not using the Chinese version lul.
> 
> I posted that just ignore. That harmless thing.

Yeah I'm aware, it's due to a few failsafes I assume, and since some things were "speed updated" as Taiga said, so I expected as much. It's just a thing I point out regardless.

**Comment by Fractune** [comment]
do I have to redownload this tool? 'cause v0.7 which I had from before is crashing my game
![image](https://user-images.githubusercontent.com/74881934/171086865-5f93f0fe-a6ea-4a8f-8276-ff8555d2ccdf.png)

**Comment by G1DE30N** [comment]
> do I have to redownload this tool? 'cause v0.7 which I had from before is crashing my game ![image](https://user-images.githubusercontent.com/74881934/171086865-5f93f0fe-a6ea-4a8f-8276-ff8555d2ccdf.png)

Yep 0.7 release dont work anymore because 2.7 update. You need build latest commit.

**Comment by G1DE30N** [comment]
MobVacuum Works (Testet on Hilichurls)

**Comment by LokeYourLord** [comment]
> > do I have to redownload this tool? 'cause v0.7 which I had from before is crashing my game ![image](https://user-images.githubusercontent.com/74881934/171086865-5f93f0fe-a6ea-4a8f-8276-ff8555d2ccdf.png)
> 
> Yep 0.7 release dont work anymore because 2.7 update. You need build latest commit.

I'm aware. I am saying **compared** to the 0.7 release. Taiga said they speed updated some things, so they probably missed changing/fixing some safety/integrity checks for 0.8 (or whatever this version will be called).

**Comment by G1DE30N** [comment]
Interactive Map Slightly Works (Icons Scrolls Strange with mouse around the screen)

**Comment by MuffinLightning** [comment]
Enabling FPS Unlock leads to crash

**Comment by G1DE30N** [comment]
> Enabling FPS Unlock leads to crash

Thanks for info, Updated post.

**Comment by G1DE30N** [comment]
Auto Tree Farm Doesn't work at all.

**Comment by G1DE30N** [comment]
Debug Menu works But any related to teleport crashes the game.

**Comment by Taiga74164** [comment]
@G1DE30N can you also send me a picture of the console output where it shows ``ProtectionBypass``

**Comment by LokeYourLord** [comment]
> @G1DE30N can you also send me a picture of the console output where it shows `ProtectionBypass`

Here: 
![image](https://user-images.githubusercontent.com/5807595/171088250-f4cccdca-273c-4539-a488-b332b652ba5a.png)

**Comment by G1DE30N** [comment]
> @G1DE30N can you also send me a picture of the console output where it shows `ProtectionBypass`

![image](https://user-images.githubusercontent.com/104440397/171088252-e2e84563-6631-49ca-8490-d02b49dc7771.png)
That you need?

**Comment by Taiga74164** [comment]
thank you @G1DE30N @LokeYourLord

**Comment by LokeYourLord** [comment]
Another crash I noticed is that the game crashes randomly after a few minutes with the message "create stackthread" at the end. I need to close the console window for the game to quit.

**Comment by G1DE30N** [comment]
> Another crash I noticed is that the game crashes randomly after a few minutes with the message "create stackthread" at the end. I need to close the console window for the game to quit. No logs I can provide though.

That strange, i in game about 30-40 minutes and except tp crashes everything fine. Need more testing and info.

**Comment by LokeYourLord** [comment]
> > Another crash I noticed is that the game crashes randomly after a few minutes with the message "create stackthread" at the end. I need to close the console window for the game to quit. No logs I can provide though.
> 
> That strange, i in game about 30-40 minutes and except tp crashes everything fine. Need more testing and info.

I turned on file logging now as well to hopefully provide more on that behalf.

**Comment by KnawKx** [comment]
can you guys commit or only callow can?

**Comment by G1DE30N** [comment]
> can you guys commit or only callow can?

As i know Only Callow can commit. But everyone can make pull requests.

**Comment by LokeYourLord** [comment]
What does work so far:
Camera Distance
Infinite Stamina (Packet Mode)
Multi-Hit (One Punch)
Auto-Loot (Normal/Unchanged Values)
Dumb Enemies
Mob Vacuum (Radius changed to 50m, distance to 3.0m, instant vacuum)
ESP
NoFog

**Comment by G1DE30N** [comment]
> What does work so far: Camera Distance Infinite Stamina (Packet Mode) Multi-Hit (One Punch) Auto-Loot (Normal/Unchanged Values) Dumb Enemies Mob Vacuum (Radius changed to 50m, distance to 3.0m, instant vacuum) ESP NoFog

Thanks, i update post immediatly. :)

**Comment by LokeYourLord** [comment]
And yes any sort of teleport, also Chest/Oculi teleport = Crash

**Comment by LokeYourLord** [comment]
> > What does work so far: Camera Distance Infinite Stamina (Packet Mode) Multi-Hit (One Punch) Auto-Loot (Normal/Unchanged Values) Dumb Enemies Mob Vacuum (Radius changed to 50m, distance to 3.0m, instant vacuum) ESP NoFog
> 
> Thanks, i update post immediatly. :)

Auto-Destroy (for Ores) also works.

**Comment by G1DE30N** [comment]
> And yes any sort of teleport, also Chest/Oculi teleport = Crash

Yep i mentioned that above

**Comment by LokeYourLord** [comment]
> > And yes any sort of teleport, also Chest/Oculi teleport = Crash
> 
> Yep i mentioned that above

Also:
Cooldown Effects:
Max Burst Energy
Reduce Skill/Burst Cooldown (0.1 change)

**Comment by G1DE30N** [comment]
> > > And yes any sort of teleport, also Chest/Oculi teleport = Crash
> > 
> > 
> > Yep i mentioned that above
> 
> Also: Cooldown Effects: Max Burst Energy Reduce Skill/Burst Cooldown (0.1 change) NoClip

Thanks, added.

**Comment by Fractune** [comment]
They may be working but that's not guaranteed they may be as safe as before, or there is?

**Comment by LokeYourLord** [comment]
Auto-Cook = Crash

**Comment by G1DE30N** [comment]
> They may be working but that's not guaranteed they may be as safe as before, or there is?

Everything always be unsafe because human factor. If you don't get impudent with values too much - everything is safe. I cheating on main acc about 1,5 months and everything fine. No Bans)

**Comment by LokeYourLord** [comment]
> > They may be working but that's not guaranteed they may be as safe as before, or there is?
> 
> Everything always be unsafe because human factor. If you don't get impudent with values too much - everything is safe. I cheating on main acc about 1,5 months and everything fine. No Bans)

I've been cheating for nearly a year, everything fine so far. Got banned once back in the early days due to not knowing about bypassing mhyprot, but once I did and used safe features and safe values, never been banned since. As long as you don't go overboard and use insane values or features you should be fine.

**Comment by BigBellyKing** [comment]
Can I get banned if I co-op with my own alt?

**Comment by G1DE30N** [comment]
> Can I get banned if I co-op with my own alt?

Nope. 100% info because my gf do that constantly about a month and not banned

**Comment by LokeYourLord** [comment]
> Can I get banned if I co-op with my own alt?

Yes, co-op is a stupid thing to cheat in regardless, because when you play in your own instance you are limited to mostly localized data. When you do co-op, you modify/alter someone else's data across servers. Stupid af to do.

**Comment by LokeYourLord** [comment]
> > Can I get banned if I co-op with my own alt?
> 
> Nope. 100% info because my gf do that constantly about a month and not banned

Yes you can, much higher risk. Don't do it.

**Comment by G1DE30N** [comment]
Ahh sorry i dont understand question well. In coop most risky thing is repirting by another people. about 60% bans in this game provided by that way as i interviewed people on forums and in the community. Subjective judgment of course, but still

**Comment by LokeYourLord** [comment]
Auto-Loot (Chests) does not work.

**Comment by G1DE30N** [comment]
> Auto-Loot (Chests)

Added

**Comment by LokeYourLord** [comment]
> Ahh sorry i dont understand question well. In coop most risky thing is reporting by another people. about 60% bans in this game provided by that way as i interviewed people on forums and in the community. Subjective judgment of course, but still
> 
> But i don't think you banned if cooped with your alt because you cannot report yourself... Or you do? :D

The chance is still much higher. It's not just about reports, it's about altering data. When you play alone, you are isolated to your own instance in which anti-cheat on the server side is active, but not as strong. You can also kind of lurk your way around while playing solo, where the anti-cheat will disregard some suspicious stuff as network or instance bugs/glitches. When you play co-op, you play on someone else instance but that instance is in a shared server now, where anti-cheat usually is much more aggressive. The chance for getting banned in co-op is just so much higher that there is no reason to play in co-op, play with your own instance and if you wanna play with your alt, use the cheat on the alt too. That is safer than co-op cheating.

**Comment by G1DE30N** [comment]
> > Ahh sorry i dont understand question well. In coop most risky thing is reporting by another people. about 60% bans in this game provided by that way as i interviewed people on forums and in the community. Subjective judgment of course, but still
> > But i don't think you banned if cooped with your alt because you cannot report yourself... Or you do? :D
> 
> The chance is still much higher. It's not just about reports, it's about altering data. When you play alone, you are isolated to your own instance in which anti-cheat on the server side is active, but not as strong. You can also kind of lurk your way around while playing solo, where the anti-cheat will disregard some suspicious stuff as network or instance bugs/glitches. When you play co-op, you play on someone else instance but that instance is in a shared server now, where anti-cheat usually is much more aggressive. The chance for getting banned in co-op is just so much higher that there is no reason to play in co-op, play with your own instance and if you wanna play with your alt, use the cheat on the alt too. That is safer than co-op cheating.

I agree BUT i cooped with my gf with cheats on my side about a week or two on her Three accounts, and noone of them will be banned.

**Comment by xuewuji** [comment]
2.7 update a "Genshin Impact\Genshin Impact game\GenshinImpact_Data\blueReporter.exe"? what this? a new mhyprot?

**Comment by LokeYourLord** [comment]
> > > Ahh sorry i dont understand question well. In coop most risky thing is reporting by another people. about 60% bans in this game provided by that way as i interviewed people on forums and in the community. Subjective judgment of course, but still
> > > But i don't think you banned if cooped with your alt because you cannot report yourself... Or you do? :D
> > 
> > 
> > The chance is still much higher. It's not just about reports, it's about altering data. When you play alone, you are isolated to your own instance in which anti-cheat on the server side is active, but not as strong. You can also kind of lurk your way around while playing solo, where the anti-cheat will disregard some suspicious stuff as network or instance bugs/glitches. When you play co-op, you play on someone else instance but that instance is in a shared server now, where anti-cheat usually is much more aggressive. The chance for getting banned in co-op is just so much higher that there is no reason to play in co-op, play with your own instance and if you wanna play with your alt, use the cheat on the alt too. That is safer than co-op cheating.
> 
> I agree BUT i cooped with my gf with cheats on my side about a week or two on her Three accounts, and noone of them will be banned.

You mean that none of them _**HAVE**_ been banned yet, not that they "will not be" banned. They certainly are at higher risk, and you are only putting yourself on their watchlist by continously doing so. On top of that, you are putting us all at risk but making cheating more obvious to them.

**Comment by Taiga74164** [comment]
> 2.7 update a "Genshin Impact\Genshin Impact game\GenshinImpact_Data\blueReporter.exe"? what this? a new mhyprot?

crashreporter

**Comment by Fractune** [comment]
They should make a menu for most safest options to use in a co-op instance

**Comment by G1DE30N** [comment]
Can you cheats in coop or not in fact it's a guessing game - no one knows for sure. But it is quite possible to light up on the video of another player, and this is a fact.

**Comment by LokeYourLord** [comment]
> They should make a menu for most safest options to use in a co-op instance

They shouldn't because co-op cheating shouldn't be something people do at all. It puts us all and the entire cheating community at risk of faster bans, patches and exposure.

**Comment by Fractune** [comment]
> > They should make a menu for most safest options to use in a co-op instance
> 
> They shouldn't because co-op cheating shouldn't be something people do at all. It puts us all and the entire cheating community at risk of faster bans, patches and exposure.

fair enough

**Comment by LokeYourLord** [comment]
> Can you cheats in coop or not in fact it's a guessing game - no one knows for sure. But it is quite possible to light up on the video of another player, and this is a fact.

I've cheated in waaaaaaaay too many games (and received my fair share of bans) from which I can gather a quite good amount of experience. It's a "guessing game" yes, but I have plenty of data to backup what I am saying (having collected 3 VAC and 2 Game bans just on Steam alone).

**Comment by ilya12121928** [comment]
and when will version 0.8 be released

**Comment by G1DE30N** [comment]
> > Can you cheats in coop or not in fact it's a guessing game - no one knows for sure. But it is quite possible to light up on the video of another player, and this is a fact.
> 
> I've cheated in waaaaaaaay too many games (and received my fair share of bans) from which I can gather a quite good amount of experience. It's a "guessing game" yes, but I have plenty of data to backup what I am saying (having collected 3 VAC and 2 Game bans just on Steam alone).

Yep you may be correct and i don't I'm not going to argue with you about this, but we have a lot of different experience because i never been banned in any game i cheat and i have one game where i cheat about a FOUR years and nobody knows about it. I'm just carrying the idea that the success of getting a ban depends not only on the technical side of the shield, but also on the person himself, including no less

**Comment by LokeYourLord** [comment]
> > > Can you cheats in coop or not in fact it's a guessing game - no one knows for sure. But it is quite possible to light up on the video of another player, and this is a fact.
> > 
> > 
> > I've cheated in waaaaaaaay too many games (and received my fair share of bans) from which I can gather a quite good amount of experience. It's a "guessing game" yes, but I have plenty of data to backup what I am saying (having collected 3 VAC and 2 Game bans just on Steam alone).
> 
> Yep you may be correct and i don't I'm not going to argue with you about this, but we have a lot of different experience because i never been banned in any game i cheat and i have one game where i cheat about a FOUR years and nobody knows about it. I'm just carrying the idea that the success of getting a ban depends not only on the technical side of the shield, but also on the person himself, including no less

You are right in saying so, but both aspects matter equally as much. And cheating in co-op is a surefire way of getting flagged eventually and leading to our all demise. I've learned as much...

**Comment by LokeYourLord** [comment]
> and when will version 0.8 be released

As @Taiga74164 said: When it's deemed stable enough. As you can see, plenty of us are currently bug testing the current build. If you wanna contribute, compile it yourself for now.

**Comment by G1DE30N** [comment]
> > > > Can you cheats in coop or not in fact it's a guessing game - no one knows for sure. But it is quite possible to light up on the video of another player, and this is a fact.
> > > 
> > > 
> > > I've cheated in waaaaaaaay too many games (and received my fair share of bans) from which I can gather a quite good amount of experience. It's a "guessing game" yes, but I have plenty of data to backup what I am saying (having collected 3 VAC and 2 Game bans just on Steam alone).
> > 
> > 
> > Yep you may be correct and i don't I'm not going to argue with you about this, but we have a lot of different experience because i never been banned in any game i cheat and i have one game where i cheat about a FOUR years and nobody knows about it. I'm just carrying the idea that the success of getting a ban depends not only on the technical side of the shield, but also on the person himself, including no less
> 
> You are right in saying so, but both aspects matter equally as much. And cheating in co-op is a surefire way of getting flagged eventually and leading to our all demise. I've learned as much...

Yes, I support your opinion that the more you climb into multiplayer, the greater the risk of being caught, if not by anti-cheat, then by other people at least.

**Comment by G1DE30N** [comment]
Added Builded Version of Latest commit for people who don't understand how build solutions.

**Comment by G1DE30N** [comment]
As i mentioned before Interactive map bugged some icons strange scrolls with mouse without logic, sometimes doesn't shows on map at all But everything fine on Minimap

**Comment by G1DE30N** [comment]
![image](https://user-images.githubusercontent.com/104440397/171094455-2d750169-b9e8-47a7-a039-76ed59bd0056.png)
Hello old friend, Apparently from now on this is an eternal problem :D

**Comment by zhohonn** [comment]
> Added Builded Version of Latest commit for people who don't understand how build solutions. @monado1110

where to find that? not on master branch

**Comment by G1DE30N** [comment]
> > Added Builded Version of Latest commit for people who don't understand how build solutions. @monado1110
> 
> where to find that? not on master branch

Link in first message in this topic)

**Comment by xuewuji** [comment]
auto talk don't work,fast dialog sitll work

**Comment by G1DE30N** [comment]
> auto talk don't work,fast dialog sitll work

Got it. Thanks.

**Comment by LokeYourLord** [comment]
> > > Added Builded Version of Latest commit for people who don't understand how build solutions. @monado1110
> > 
> > 
> > where to find that? not on master branch
> 
> Link in first message in this topic)

Personally, those who don't know how to build should also not get a link to a finished build from you, as the program is still buggy and crashy and there will be complains about "wHy IsN't It StAbLe". Also, they risk getting banned more easier or other side affects. I would not provide that link but it is up to you.

**Comment by Equ1no0x** [comment]
> Personally, those who don't know how to build should also not get a link to a finished build from you, as the program is still buggy and crashy and there will be complains about "wHy IsN't It StAbLe". Also, they risk getting banned more easier or other side effects. I would not provide that link but it is up to you.

Sure, I can see this happening, but at the same time, more ppl using it, more feedback, even if some of the feedback is nonsense.

**Comment by G1DE30N** [comment]
I share it simply for speeding up bug reporting, i dont mention to provide people to more bans, if you afraid for your acc, you can simply ignore this link.

**Comment by Legend-Recalls** [comment]
> > > > Can you cheats in coop or not in fact it's a guessing game - no one knows for sure. But it is quite possible to light up on the video of another player, and this is a fact.
> > > 
> > > 
> > > I've cheated in waaaaaaaay too many games (and received my fair share of bans) from which I can gather a quite good amount of experience. It's a "guessing game" yes, but I have plenty of data to backup what I am saying (having collected 3 VAC and 2 Game bans just on Steam alone).
> > 
> > 
> > Yep you may be correct and i don't I'm not going to argue with you about this, but we have a lot of different experience because i never been banned in any game i cheat and i have one game where i cheat about a FOUR years and nobody knows about it. I'm just carrying the idea that the success of getting a ban depends not only on the technical side of the shield, but also on the person himself, including no less
> 
> You are right in saying so, but both aspects matter equally as much. And cheating in co-op is a surefire way of getting flagged eventually and leading to our all demise. I've learned as much...

lmao leading to our demise damn, its an open source cheat, honestly they can start flagging us whenever they want, coop is safe af I been cheating for like 4 months

**Comment by G1DE30N** [comment]
> Xiao's E doesn't do any damage when multi hit is on. Will mention this on #10 also.

Yep, sorry i don't add to this list because its specific characters bug instead of globals mentioned here.

**Comment by souvikpodder** [comment]
> Yep, sorry i don't add to this list because its specific characters bug instead of globals mentioned here.

That's okay. I mentioned the issue in #10. Do you know if it was working before 2.7?

**Comment by G1DE30N** [comment]
> > Yep, sorry i don't add to this list because its specific characters bug instead of globals mentioned here.
> 
> That's okay. I mentioned the issue in #10. Do you know if it was working before 2.7?

Unfortunately, no, because I don't use charged attacks at all in my gameplay style, and I don't have xiao in sight

**Comment by G1DE30N** [comment]
@CallowBlack i described all collected problems in first post. You can see them now.

**Comment by CallowBlack** [comment]
> > @G1DE30N Can you sum all messages up in initial message. Because I am too lazy to read 80+ messages, thanks. Also, can you note what feature causes crash.
> 
> But i do that already..... * confuse *.

Yeah, I saw it a little after. Thanks.

**Comment by Fractune** [comment]
FYI teleport didn't crash my game. It just did not work

**Comment by souvikpodder** [comment]
> Need more info from others, because for me personally thats is instacrash. And most strange because for me console don't trows any errors, it simple shows stage 0 and 1 from teleporting script and then game silently closes.

for me also teleport using "T" hotkey doesn't crash the game. It doesn't do anything.

**Comment by Legend-Recalls** [comment]
> > FYI teleport didn't crash my game. It just did not work (err on console) and that's about it.
> 
> Need more info from others, because for me personally thats is instacrash. And most strange because for me console don't trows any errors, it simple shows stage 0 and 1 from teleporting script and then game silently closes.

^^^  samee

**Comment by CNLouisLiu** [comment]
maybe something went wrong with TO_UNI_DICT,because, when the waypoints is teleported, the code runs here and the game will exit

**Comment by CallowBlack** [comment]
@G1DE30N Can you describe what is wrong with `No clip`, because for me all works correctly.

**Comment by G1DE30N** [comment]
> @G1DE30N Can you describe what is wrong with `No clip`, because for me everything works correctly.

When you active "No Animation" in active no clip Character Animations WONT stops.

**Comment by G1DE30N** [comment]
> maybe something went wrong with TO_UNI_DICT,because, when the waypoints is teleported, the code runs here and the game will exit. Also, the game will be exited here during AutoTreeFarm, and the console will receive a warning error.

If i turn on auto tree farm this func simple doesn't do anything at all, wood not increases.

**Comment by CallowBlack** [comment]
> > @G1DE30N Can you describe what is wrong with `No clip`, because for me everything works correctly.
> 
> When you active "No Animation" in active no clip Character Animations WONT stops.

Are you sure with it? 
![image](https://user-images.githubusercontent.com/37184274/171147288-cf0854c2-0a6c-458b-bf7e-605b11620b72.png)
Maybe I fixed some related bug previously. But would be good if you recheck it, of course, if you can.

**Comment by CNLouisLiu** [comment]
> > maybe something went wrong with TO_UNI_DICT,because, when the waypoints is teleported, the code runs here and the game will exit. Also, the game will be exited here during AutoTreeFarm, and the console will receive a warning error.
> 
> If i turn on auto tree farm this func simple doesn't do anything at all, wood not increases.
You will see an error in the console like
![image](https://user-images.githubusercontent.com/35774374/171148229-edf517b4-1bf7-4df5-92dc-04d0392b93df.png)

**Comment by G1DE30N** [comment]
> > > maybe something went wrong with TO_UNI_DICT,because, when the waypoints is teleported, the code runs here and the game will exit. Also, the game will be exited here during AutoTreeFarm, and the console will receive a warning error.
> > 
> > 
> > If i turn on auto tree farm this func simple doesn't do anything at all, wood not increases.
> > You will see an error in the console like
> > ![image](https://user-images.githubusercontent.com/35774374/171148229-edf517b4-1bf7-4df5-92dc-04d0392b93df.png)

Nope, Console still silent. the game behaves as if nothing has changed after switching on.

**Comment by CallowBlack** [comment]
Oh, yeah they update .NET Framework to 4.0, so Dictionary structure was changed.
Thanks for observation.

**Comment by CallowBlack** [comment]
> > > > @G1DE30N Can you describe what is wrong with `No clip`, because for me everything works correctly.
> > > 
> > > 
> > > When you active "No Animation" in active no clip Character Animations WONT stops.
> > 
> > 
> > Are you sure with it? ![image](https://user-images.githubusercontent.com/37184274/171147288-cf0854c2-0a6c-458b-bf7e-605b11620b72.png) Maybe I fixed some related bug previously. But would be good if you recheck it, of course, if you can.
> 
> Maybe i cannot understand how it works? It func stop animation when you turn on noclip or blocks any animations while noclip us active? I check it rn pls wait.
> 
> If first - if i turn on noclip in sprint - Character loked in sprint animation after noclip is turned. If second - in noclip idle animations still played.

If you enable NoClip with enabled `No animations` that animation will not change after enabling, not all, attack animation still work. But if it is disabled, Jump, Run and everything working.

**Comment by CNLouisLiu** [comment]
> ![image](https://user-images.githubusercontent.com/104440397/171149874-1a25bbbb-12ff-42ce-a448-d3dd7373d404.png) Kill aura wont work - confirmed. I tried with different mobs, and nothing. both modes. No errors in console.
Please try DamageMode, my DamageMode is working fine

**Comment by CNLouisLiu** [comment]
![image](https://user-images.githubusercontent.com/35774374/171150436-2bd7abc5-f199-4c29-8e11-61ad5231e91c.png)
He successfully killed the monster

**Comment by G1DE30N** [comment]
> > ![image](https://user-images.githubusercontent.com/104440397/171149874-1a25bbbb-12ff-42ce-a448-d3dd7373d404.png) Kill aura wont work - confirmed. I tried with different mobs, and nothing. both modes. No errors in console.
> 
> Please try DamageMode, my DamageMode is working fine

Where damage mode? 
![image](https://user-images.githubusercontent.com/104440397/171150600-d0e08042-3778-4138-9a8f-7782b124f7bb.png)

**Comment by CNLouisLiu** [comment]
> > > ![image](https://user-images.githubusercontent.com/104440397/171149874-1a25bbbb-12ff-42ce-a448-d3dd7373d404.png) Kill aura wont work - confirmed. I tried with different mobs, and nothing. both modes. No errors in console.
> > 
> > 
> > Please try DamageMode, my DamageMode is working fine
> 
> Where damage mode? ![image](https://user-images.githubusercontent.com/104440397/171150600-d0e08042-3778-4138-9a8f-7782b124f7bb.png) You mean crash? i always use crash, and for me crash and instadeath not work at all.

sorry is InstantDeathMode

**Comment by G1DE30N** [comment]
UPD: For me kill aura work in a really strange way.
It work only for tick when i turn on or off it.
I mean - When i turn on and off - in crash mode mobs get 2 strikes.
In instadeath mobs dies only after i tick turn on. While Kill aura remains active - nothing happened

**Comment by CNLouisLiu** [comment]
> UPD: For me kill aura work in a really strange way. It work only for tick when i turn on or off it. I mean - When i turn off - in crash mode mobs get 1 strike and nothing happened after. In instadeath mobs dies only after i tick turn off. While Kill aura remains active - nothing happened

He also failed when I entered 2.7 for the first time. After I hit the monster, it worked, and it kept working. maybe you can try

**Comment by G1DE30N** [comment]
> > UPD: For me kill aura work in a really strange way. It work only for tick when i turn on or off it. I mean - When i turn off - in crash mode mobs get 1 strike and nothing happened after. In instadeath mobs dies only after i tick turn off. While Kill aura remains active - nothing happened
> 
> He also failed when I entered 2.7 for the first time. After I hit the monster, it worked, and it kept working. maybe you can try

I tried and nope. Doesn't helps.

**Comment by G1DE30N** [comment]
When turn off auto tree farm console get infinite loop of this 
![image](https://user-images.githubusercontent.com/104440397/171152386-bf250292-fc25-4586-a5f6-cd765e8f4d61.png)

**Comment by CNLouisLiu** [comment]
> When turn on auto tree farm console get infinite loop of this ![image](https://user-images.githubusercontent.com/104440397/171152386-bf250292-fc25-4586-a5f6-cd765e8f4d61.png)

Callow said above, genshen upgraded .net4.0 ,Dictionary structure was changed.

**Comment by G1DE30N** [comment]
> > When turn on auto tree farm console get infinite loop of this ![image](https://user-images.githubusercontent.com/104440397/171152386-bf250292-fc25-4586-a5f6-cd765e8f4d61.png)
> 
> Callow said above, genshen upgraded .net4.0 ,Dictionary structure was changed.

Ah okay.

**Comment by G1DE30N** [comment]
After some relloging Instant death killaura works, but crash mode doesn't no matter what i try.

**Comment by tedzify** [comment]
`No animation` in `NoClip` means if you control your character for walking and jumping it won't show that animation, and if you turn on 'NoClip' with 'No Animation' while your character is in an animation, e.g. swimming, falling, running, it will preserve that animation until you turn off 'NoClip". If 'No Animation' is `disabled`, your character can walk, jump, etc. while using 'NoClip' controls. That's just how it works, the updated didn't change it.

**Comment by G1DE30N** [comment]
> `No animation` in `NoClip` means if you control your character for walking and jumping it won't show that animation, and if you turn on 'NoClip' with 'No Animation' while your character is in an animation, e.g. swimming, falling, running, it will preserve that animation until you turn off 'NoClip". If 'No Animation' is `disabled`, your character can walk, jump, etc. while using 'NoClip' controls. That's just how it works, the updated didn't change it.

But as i remember noclip works like this as well in old releases before that "no animation thing" added, especially for controls. I don't remember times where character can walk (run animation) in midair while noclip. Hmmm.

**Comment by tedzify** [comment]
Issue with `Auto Cook`, for the recipes which are not completed even if `Count Item = 1`, it would still complete it **visually** (the proficiency count resets after teleport). 

How to replicate:
1. Turn on `Auto Cook`, set `Count Item` to 1 (default)
2. Cook a food that is still not 100% completed in proficiency
3. Watch the proficiency quickly rise. e.g. proficiency 0/25 quickly goes to 25/25

This is just visual, doesn't increase `Master Chef` achievement.

**Comment by tedzify** [comment]
> > `No animation` in `NoClip` means if you control your character for walking and jumping it won't show that animation, and if you turn on 'NoClip' with 'No Animation' while your character is in an animation, e.g. swimming, falling, running, it will preserve that animation until you turn off 'NoClip". If 'No Animation' is `disabled`, your character can walk, jump, etc. while using 'NoClip' controls. That's just how it works, the updated didn't change it.
> 
> But as i remember noclip works like this as well in old releases before that "no animation thing" added, especially for controls. I don't remember times where character can walk (run animation) in midair while move in noclip. Hmmm. If it added to player can animated in noclip for some reason, okay. But why.

My guess would be 'No Animation' freezes the current animation of the character.

**Comment by LokeYourLord** [comment]
> Issue with `Auto Cook`, for the recipes which are not completed even if `Count Item = 1`, it would still complete it **visually** (the proficiency count resets after teleport).
> 
> How to replicate:
> 
>     1. Turn on `Auto Cook`, set `Count Item` to 1 (default)
> 
>     2. Cook a food that is still not 100% completed in proficiency
> 
>     3. Watch the proficiency quickly rise. e.g. proficiency 0/25 quickly goes to 25/25
> 
> 
> This is just visual, doesn't increase `Master Chef` achievement.

I've mentioned the issue before, but for me it crashes outright.

**Comment by tedzify** [comment]
> I've mentioned the issue before, but for me it crashes outright.

Only `Map Teleport` crashes my game, the teleport menu just doesn't work.

**Comment by CNLouisLiu** [comment]
Sinubukan kong itakda ang teleport waypoint sa isang malapit na waypoint, ngunit ang MoleMole_LocalEntityInfoData_get_initPos Hook ay mukhang hindi magkakabisa at ang proseso ng teleport ay hindi nagpapatuloy.

**Comment by LokeYourLord** [comment]
> Sinubukan kong itakda ang teleport waypoint sa isang malapit na waypoint, ngunit ang MoleMole_LocalEntityInfoData_get_initPos Hook ay mukhang hindi magkakabisa at ang proseso ng teleport ay hindi nagpapatuloy.

Have you noticed everyone here speaks in English even though the majority of us either speak additional languages or have another mother-tongue but still speak English?...

**Comment by tedzify** [comment]
> Thats normal if you try auto cook item without mastery, because cheat utilizes mastery game mechanic, this not a bug in fact.

It doesn't make sense that it isn't a bug, because it specifically says this (I'm assuming setting it to 1 would also give 1 proficiency, cause the cheat making it jump instantly to max is definitely a bug):
![Screenshot_98](https://user-images.githubusercontent.com/99535026/171162263-9e7f259f-7ce0-4dc0-8345-33c4a60e68bd.png)

**Comment by LokeYourLord** [comment]
> > Thats normal if you try auto cook item without mastery, because cheat utilizes mastery game mechanic, this not a bug in fact.
> 
> It doesn't make sense that it isn't a bug, because it specifically says this (I'm assuming setting it to 1 would also give 1 proficiency, cause the cheat is making it jump instantly to max is definitely a bug): ![Screenshot_98](https://user-images.githubusercontent.com/99535026/171162263-9e7f259f-7ce0-4dc0-8345-33c4a60e68bd.png)

That's exactly what I am talking about as well, it can auto-finish unlearned/unmastered recipes for you (as it is what is said in the text bubble). So Auto-Cook is just broken at the moment.

**Comment by tedzify** [comment]
> That's exactly what I am talking about as well, it can auto-finish unlearned/unmastered recipes for you (as it is what is said in the text bubble). So Auto-Cook is just broken at the moment.

Seem like `Auto Cook` needed some more work even before the update, just read the notes on the commit.

**Comment by Joni4H** [comment]
![unknown](https://user-images.githubusercontent.com/106591520/171167427-cdac070a-352f-4159-9a68-436800c54d2b.png)
What to do with it?

**Comment by tedzify** [comment]
> ![unknown](https://user-images.githubusercontent.com/106591520/171167427-cdac070a-352f-4159-9a68-436800c54d2b.png) What to do with it?

Ignore it.

**Comment by Joni4H** [comment]
> > ![неизвестный](https://user-images.githubusercontent.com/106591520/171167427-cdac070a-352f-4159-9a68-436800c54d2b.png)Что с этим делать?
> 
> Игнорируй это. На впрыск никак не влияет.

After that it crashes the game and that's it.

**Comment by RyujinZX** [comment]
Auto Treasure - fixed
Auto Talk - fixed
FPS Unlock - fixed

**Comment by Kolya62** [comment]
At the entrance it gives an error: 

upload_crash release upload: 398598
200
file upload finish

And then the game crashes

**Comment by Joni4H** [comment]
> Вам нужно собрать фиксацию.

How to build the latest commit?

**Comment by Kolya62** [comment]
then the game crashes

![Снимок экрана (8)](https://user-images.githubusercontent.com/106597499/171172495-e4654beb-01b8-47b2-81f3-05f619bd88bd.png)

how to fix it

**Comment by LokeYourLord** [comment]
> then the game crashes
> 
> ![Снимок экрана (8)](https://user-images.githubusercontent.com/106597499/171172495-e4654beb-01b8-47b2-81f3-05f619bd88bd.png)
> 
> how to fix it



> upload_crash release upload:351133 200 file upload finish then the game crashes ![image](https://user-images.githubusercontent.com/106572945/171171832-32b250eb-dda6-4121-9ba3-1e6a93613bbc.png) how to fix it

You both are using an outdated version, maybe you could read for a second? It says "Seems like assembly checksum don't match with version **_2.6_**."

The update for 2.7 has not yet been released, please learn to read for about 2 seconds.

**Comment by RyujinZX** [comment]
> then the game crashes
> 
> ![Снимок экрана (8)](https://user-images.githubusercontent.com/106597499/171172495-e4654beb-01b8-47b2-81f3-05f619bd88bd.png)
> 
> how to fix it

You run an old version of the reader, you need to compile the current build, or wait for the release.

**Comment by keexers** [comment]
> > then the game crashes
> > ![Снимок экрана (8)](https://user-images.githubusercontent.com/106597499/171172495-e4654beb-01b8-47b2-81f3-05f619bd88bd.png)
> > how to fix it
> 
> > upload_crash release upload:351133 200 file upload finish then the game crashes ![image](https://user-images.githubusercontent.com/106572945/171171832-32b250eb-dda6-4121-9ba3-1e6a93613bbc.png) how to fix it
> 
> You both are using an outdated version, maybe you could read for a second? It says "Seems like assembly checksum don't match with version **_2.6_**."
> 
> The update for 2.7 has not yet been released, please learn to read for about 2 seconds @Kolya62 @hattoya

the update for 2.7 is released but its not that stable yet, you need to compile it manually

**Comment by LokeYourLord** [comment]
> > > then the game crashes
> > > ![Снимок экрана (8)](https://user-images.githubusercontent.com/106597499/171172495-e4654beb-01b8-47b2-81f3-05f619bd88bd.png)
> > > how to fix it
> > 
> > 
> > > upload_crash release upload:351133 200 file upload finish then the game crashes ![image](https://user-images.githubusercontent.com/106572945/171171832-32b250eb-dda6-4121-9ba3-1e6a93613bbc.png) how to fix it
> > 
> > 
> > You both are using an outdated version, maybe you could read for a second? It says "Seems like assembly checksum don't match with version **_2.6_**."
> > The update for 2.7 has not yet been released, please learn to read for about 2 seconds @Kolya62 @hattoya
> 
> the update for 2.7 is released but its not that stable yet, you need to compile it manually

Ok since we are getting "technically correct" here, I said it hasn't been released, which is true. _**Officially**_ it hasn't been released, but the debug/"current build" is there for anyone to compile. But a final release build/public release build has not been released yet, is what I obviously meant to say with that ^-^

**Comment by tedzify** [comment]
> After that it crashes the game and that's it.

Compile the latest commit yourself, the `Release v0.7` is not compatible with Genshin 2.7.

**Comment by keexers** [comment]
> > > > then the game crashes
> > > > ![Снимок экрана (8)](https://user-images.githubusercontent.com/106597499/171172495-e4654beb-01b8-47b2-81f3-05f619bd88bd.png)
> > > > how to fix it
> > > 
> > > 
> > > > upload_crash release upload:351133 200 file upload finish then the game crashes ![image](https://user-images.githubusercontent.com/106572945/171171832-32b250eb-dda6-4121-9ba3-1e6a93613bbc.png) how to fix it
> > > 
> > > 
> > > You both are using an outdated version, maybe you could read for a second? It says "Seems like assembly checksum don't match with version **_2.6_**."
> > > The update for 2.7 has not yet been released, please learn to read for about 2 seconds @Kolya62 @hattoya
> > 
> > 
> > the update for 2.7 is released but its not that stable yet, you need to compile it manually
> 
> Ok since we are getting "technically correct" here, I said it hasn't been released, which is true. _**Officially**_ it hasn't been released, but the debug/"current build" is there for anyone to compile. But a final release build/public release build has not been released yet, is what I obviously meant to say with that ^-^

oh okay, sorry for my nonsense

**Comment by hattoya** [comment]
> > After that it crashes the game and that's it.
> 
> Compile the latest commit yourself, the `Release v0.7` is not compatible with Genshin 2.7.

I would love to do it if I could..

**Comment by s2120205** [comment]
> **Bug Table**
> 
> Player:
> 
> * ~Godmode - Works for block Fall Damage but doesn't work for all other sources of damage (mob attacks, damage traps, effects of elements etc.~
> * ~Noclip - Most of all works fine but NoAnimation Func doesn't do anything at all.~
> 
> World:
> 
> * ~Auto Treasure - Wont open chests.~
> * ~Auto Talk - Auto Select dialog doesn't work.~
> * ~Auto Tree Farm - Doesn't Work Entirely.~
> * ~Auto Cook  - Doesn't Work Entirely.~
> * ~Kill Aura - Doesn't Work Entirely.~
> 
> Teleport:
> 
> * ~Any type of Teleporting Crashes the game. (Or for someone simple won't work at all)~
> 
> Visuals: ~FPS Unlock - Crashes the game.~
> 
> Interactive Map: ~- Doesn't work entirely except radar, alll icons messed up and move on screen while you scroll map with cursor. Sometimes icons doesn't shows at all.~
> 
> ==================================================
> 
> MAYBE ONLY FOR ME: (@CallowBlack, Just a note for info, Don't focus your attention too much on this, because so far there is no confirmation from other players here. )
> 
> * UI messed up entirely immediately after starting the game with no matter what cheat funcs active or not. The game stupidly starts immediately with a broken interface. I thinks devs f**cked up render engine in new update even stronger than in the previous version. Perhaps that's why the interactive map broke down. But again, these are all my theories, in general I'm thinking that most likely the gui library itself that you use is the root of this evil. So many mysteries.
> 
> ==================================================
> 
> **Here are not tested for functionality features:**
> 
> Player:
> 
> * ~Attack Effects (_Multi-Target_, _Radius_)~
> * ~Cooldown Effects (_No Sprint Cooldown_, _Instant Bow Charge_)~
> 
> World:
> 
> * ~Auto Destroy Objects (_Shields_, _Doodads_)~
> * ~Auto Treasure (_Leyline_, _Search Points_, _Quest Interacts_, _Others_)~
> * Fishing (_Delay_, _Recast Rod_, _Recast Delay_)
> * ~Mob Vacuum (_Animals_)~
> 
> **Potentialy harmless (as Taiga said):** A bunch of error codes in startup and reminder about chinesse client.
> 
> **_If you find anything except this list, please, provide it below and i add that info there. Thats helps to Callow, Taiga and other contributors fix this ASAP._**

Hello iam new here is the rapid fire or multi hit detectable?

**Comment by rathnesh12** [comment]
my UI too sometimes get tangled up
for example when i have to use a food on a chara the other buttons block the top row of list
it also occours while i view artifacts stats 
i forgot to take a screenshot 
but after i disabled a feature the ui was back to normal 
and the sad part is i am not clear which feature it was
